src/groups/general_linear.jl

```diff
@@ -105,50 +109,62 @@ function exp!(G::GeneralLinear{2}, q, p, X)
 end
 
 function get_coordinates(
-    ::GeneralLinear{n,ℝ},
+    ::GeneralLinear{<:Any,ℝ},
     p,
     X,
     ::DefaultOrthonormalBasis{ℝ,TangentSpaceType},
-) where {n}
+)
     return vec(X)
 end
 
 function get_coordinates!(
-    ::GeneralLinear{n,ℝ},
+    ::GeneralLinear{<:Any,ℝ},
     Xⁱ,
     p,
     X,
     ::DefaultOrthonormalBasis{ℝ,TangentSpaceType},
-) where {n}
+)
     return copyto!(Xⁱ, X)
 end
 
-get_embedding(::GeneralLinear{n,𝔽}) where {n,𝔽} = Euclidean(n, n; field=𝔽)
+function get_embedding(::GeneralLinear{TypeParameter{Tuple{n}},𝔽}) where {n,𝔽}
+    return Euclidean(n, n; field=𝔽, parameter=:type)
+end
+function get_embedding(M::GeneralLinear{Tuple{Int},𝔽}) where {𝔽}
```

Comment:
We could have only one method of `get_embedding` per manifold but then we would have to delegate dispatch on parameter to another function with two methods. It would make sense if that function had other uses but I haven't found them yet.